### PR TITLE
LTT modpack + local server file

### PR DESCRIPTION
**Pull request by jwrober**

### CODE REVIEW

File: LTT/LTT_local.serv [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+set techlossforgiveness -1
+set techlossrestore 50
+set freecost 50
+set diplcost 50
+set conquercost 50
+
+# technical
+set aifill=10
+set allowtake=""
+cmdlevel info
```
Comment (pranavtifr) [inline LTT/LTT_local.serv:70]: Comment this out too. Local players can have edit level access.

File: LTT/LTT_local.serv [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+set ec_chat=enabled
+set ec_info=enabled 
+set ec_max_size=5000
+set ec_turns=9
+set maxplayers=126
+set saveturns=1
+set scorelog=enabled
+set timeout=0
+set unitwaittime=36000
+set fixedlength enabled
```
Comment (lmoureaux) [inline LTT/LTT_local.serv:81]: Delete this for local games

File: LTT/LTT_local.serv [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+#set nuke_improvements = 20		# 20% chance for destroying city improvements. never destroyed if the sabotage chance is less than 100%
+
+# for testing purposes now, check for the actual game
+#set unitwaittime_range = -1
+#set unitwaittime_allied = enabled
+#set unitwaittime_extended = enabled
+#set playerwaittime = 0
+
+metamessage "Longturn Traditional for multiplayer games | More info: http://longturn.net"
+metapatches longturn
```
Comment (lmoureaux) [inline LTT/LTT_local.serv:102]: Delete this too

File: LTT/LTT_local.serv [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+set compresstype plain
+set ec_chat=enabled
+set ec_info=enabled 
+set ec_max_size=5000
+set ec_turns=9
+set maxplayers=126
+set saveturns=1
+set scorelog=enabled
+set timeout=0
+set unitwaittime=36000
```
Comment (pranavtifr) [inline LTT/LTT_local.serv:80]: No need for UWT in local games.

File: LTT/LTT_local.serv [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+set aifill=10
+set allowtake=""
+#cmdlevel info # Allows for hack level in local games
+set compresstype plain
+set ec_chat=enabled
+set ec_info=enabled 
+set ec_max_size=5000
+set ec_turns=9
+set maxplayers=126
+set saveturns=1
```
Comment (pranavtifr) [inline LTT/LTT_local.serv:77]: Not really sure if this and scorelog is needed either. They pollute folders. Player can enable if needed before the game begins.

**Review by lmoureaux — APPROVED** [review]

**Pull request merged**